plutus-core/cost-model/budgeting-bench/Common.hs

```diff
@@ -92,38 +92,42 @@ benchDefault = benchWith defaultCekParameters
 
 ---------------- Constructing Polymorphic PLC terms for benchmarking ----------------
 
-integer :: uni `Includes` Integer => Type tyname uni ()
+integer :: uni `HasTypeLevel` Integer => Type TyName uni ()
 integer = mkTyBuiltin @_ @Integer ()
 
-bytestring :: uni `Includes` BS.ByteString => Type tyname uni ()
+bytestring :: uni `HasTypeLevel` BS.ByteString => Type TyName uni ()
 bytestring = mkTyBuiltin @_ @BS.ByteString ()
 
 
 -- To make monomorphic terms, make tys equal to [] in the mkApp functions
 
 -- Just make the term (con unit ()), which is about the simplest possible.
-mkUnit :: uni `Includes` () => PlainTerm uni fun
+mkUnit :: uni `HasTermLevel` () => PlainTerm uni fun
```

Comment:
> I think if you write `HasBothLevel` when you only really need `HasTermLevel` (like here, for example) then there won't be an error/warning, right? 

Unfortunately, right. No idea on how to improve on this.

> We should probably try to be as generic as we can when we're writing these constraints though since that might make things a bit more understandable.

I agree.